src/Discord.Net.WebSocket/DiscordSocketClient.cs

```diff
@@ -1270,13 +1270,17 @@ private async Task ProcessMessageAsync(GatewayOpCode opCode, int? seq, string ty
                                             return;
                                         }
 
+                                        var cacheablelist = new List<Cacheable<IMessage, ulong>>();
                                         foreach (ulong id in data.Ids)
                                         {
                                             var msg = SocketChannelHelper.RemoveMessage(channel, this, id);
                                             bool isCached = msg != null;
                                             var cacheable = new Cacheable<IMessage, ulong>(msg, id, isCached, async () => await channel.GetMessageAsync(id));
-                                            await TimedInvokeAsync(_messageDeletedEvent, nameof(MessageDeleted), cacheable, channel).ConfigureAwait(false);
```

Comment:
I'm not sure if we should remove this; I could see reasons for still wanting to capture single message deletes when a bulk delete occurs.

Reply:
You could still do that as it returns a list of IMessages